sdk/src/time_stamp.rs

```diff
@@ -117,8 +118,15 @@ fn time_stamp_request_http(
 
     let body_reader = std::io::Cursor::new(body);
 
-    let response = ureq::post(url)
-        .set("Content-Type", HTTP_CONTENT_TYPE_REQUEST)
+    let mut req = ureq::post(url).set("Content-Type", HTTP_CONTENT_TYPE_REQUEST);
```

Comment:
Maybe we should set the “Content-Type” header after we've added the user provided headers, so we can prevent a user from breaking the spec by providing their own ContentType header?

Reply:
The content type for timestamp servers must be "application/timestamp-query"

Reply:
Similarly the response type must be: "application/timestamp-reply"

Reply:
@mauricefisher64 I moved the request header to after the user-specified headers per @dyro's suggestion so that it will overwrite any user-specified `Content-Type` header. At line 111 of this file, we define ...

```
    const HTTP_CONTENT_TYPE_REQUEST: &str = "application/timestamp-query";
```

.. so the request should be formatted as you say.